cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppFabricHttpHandler.java

```diff
@@ -1052,10 +1054,9 @@ public boolean apply(Id.Program programId) {
       appLifecycleHttpHandler.deleteMetrics(account, null);
       // delete all meta data
       store.removeAll(namespaceId);
-      // todo: delete only for specified account
       // delete queues and streams data
-      queueAdmin.dropAll();
-      streamAdmin.dropAll();
+      queueAdmin.dropAllInNamespace(namespaceId.getId());
```

Comment:
For consistency, can you add a TODO to change the signature to ``dropAllInNamespace(Id.Namespace)`` and create a bug?